packages/desktop-client/src/components/accounts/Filters.js

```diff
@@ -200,6 +202,16 @@ function ConfigureField({
           )}
         </View>
 
+        <View
+          style={{
+            color: colors.n4,
+            marginBottom: 10,
+            alignItems: 'left',
```

Comment:
I don’t think this is a valid CSS value — would recommend trying `P` if you haven’t already here.

Reply:
Sorry, I can't tell which value you are referencing.

Reply:
The `left` value on `alignItems`